local-proxy/src/test/test_flakyInvoke.ts

```diff
@@ -0,0 +1,32 @@
+import * as http from 'http';
+import { AddressInfo } from 'net';
+import express from 'express';
+import { setupProxy } from '../index';
+import test from 'ava';
+import path from 'path';
+import got from 'got';
+
+// This test should be considered NOT flaky
+// This means that if a test run fails there is most likely a bug in the code
+test('Flaky http proxy', async (t) => {
```

Comment:
What does this test?  AFAICT the proxy itself _isn't_ flaky here; could you retitle or comment?